packages/core/CHANGELOG.md

```diff
@@ -5,32 +5,21 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ## [2.45.1](https://github.com/carbon-design-system/carbon-components-vue/compare/@carbon/vue@2.45.0...@carbon/vue@2.45.1) (2023-01-24)
 
-
```

Comment:
Can you revert the changes to this file. I think they are unintentional.
```sh
git checkout 615311b8faab77997f768f38acee3d9a2a5e57dc packages/core/CHANGELOG.md
git add packages/core/CHANGELOG.md
git commit
```

Reply:
[77eb4c8](https://github.com/carbon-design-system/carbon-components-vue/pull/1420/commits/77eb4c892996e3cd0c3d0160ce7522df461d7008)